setup.py

```diff
@@ -106,23 +106,28 @@ def setup_op_geth_repo():
 
 def setup_blockscout_repo():
     github_url = "https://github.com/blockscout/blockscout.git"
-    git_tag = "49eee16ee1078a975a060b1564e6ea5b1ad70f39"
+    git_tag = "b6136977051730a7acd47b35b6bdcf9a74e39be9"
 
     if os.path.isfile("blockscout"):
         raise Exception("Error: 'blockscout' exists as a file and not a directory.")
     elif not os.path.exists("blockscout"):
         print("Cloning the blockscout repository. This may take a while...")
         lib.clone_repo(github_url, "clone the blockscout repository")
 
-        lib.run("checkout stable version", f"git checkout --detach {git_tag}",
+        lib.run("checkout optimism stable version", f"git checkout --detach {git_tag}",
                 cwd="blockscout")
 
         # TODO make this not replace multiple times if run multiple times
+        anchor_line = "image: blockscout/blockscout:${DOCKER_TAG:-latest}"
```

Comment:
Could you also take care of the TODO above, if it isn't already the case?

Reply:
it still used, the L126 replace it with a specific optimism image which i tested works.

Reply:
i think i didn't try to fix the TODO case